src/context/FoxEthProvider/FoxEthProvider.tsx

```diff
@@ -140,6 +140,7 @@ const FoxLpAndFarmingOpportunitiesContext = createContext<IFoxLpAndFarmingOpport
 })
 
 export const FoxEthProvider = ({ children }: FoxEthProviderProps) => {
+  const featureFlags = useAppSelector(selectFeatureFlags)
```

Comment:
We might want to destructure `FoxLP` and `FoxFarming` from here.

Reply:
Will tackle in a follow-up smallish PR so this can get in